roles/openshift_service_catalog/files/kubeservicecatalog_roles_bindings.yml

```diff
@@ -7,12 +7,12 @@ objects:
 - kind: ClusterRole
   apiVersion: v1
   metadata:
-    name: servicecatalog-serviceclass-viewer
+    name: servicecatalog-clusterserviceclass-viewer
   rules:
   - apiGroups:
     - servicecatalog.k8s.io
     resources:
-    - serviceclasses
+    - clusterserviceclasses
```

Comment:
This role should also have permission to view the clusterserviceplans resource; see: https://github.com/openshift/origin/blob/master/examples/service-catalog/service-catalog.yaml#L7

You should make sure, as far as I'm aware, that these two files match.

Eventually I would like to not have this content duplicated, if we can figure out the right way to do that, see #5039